elyra/cli/pipeline_app.py

```diff
@@ -149,6 +151,42 @@ def _preprocess_pipeline(pipeline_path: str, runtime: str, runtime_config: str)
     return pipeline_definition
 
 
+def _print_issues(issues):
+    # print validation issues
+    for issue in issues:
+        if issue.get('severity') == ValidationSeverity.Error:
+            click.echo(
+                f'- (Fatal error on node \'{issue["data"]["nodeName"]}\' property \'{issue["data"]["propertyName"]}\') '
+                f'- {issue["message"]}')
+        else:
+            # TODO check warning nodeNames are empty
+            click.echo(
+                f'- (Warning) - {issue["message"]}')
```

Comment:
There are several severity levels.  The severity level name can be printed using the enum's built-in `name` property: `issue.get('severity').name`.